dev/ParallaxView/APITests/ParallaxViewTests.cs

```diff
@@ -1433,7 +1433,7 @@ public void VerifyParallaxViewOffsetChangeInsideScrollViewer()
             scrollViewer.Height = c_defaultUIScrollViewerHeight;
             scrollViewer.HorizontalScrollBarVisibility = ScrollBarVisibility.Hidden;
             scrollViewer.VerticalScrollBarVisibility = ScrollBarVisibility.Hidden;
-            scrollViewer.ZoomMode = ZoomMode.Disabled;
+            scrollViewer.ZoomMode = Windows.UI.Xaml.Controls.ZoomMode.Disabled;
```

Comment:
nit: fully qualified here but not below.

Reply:
This was done to disambiguate between the WUX and MUX versions of the enum.